src/ralph/cli.py

```diff
@@ -393,3 +397,52 @@ def list_(details, new, backend, **options):
 
     if counter == 0:
         logger.warning("Configured %s backend contains no archive", backend)
+
+
+@cli.command()
+@click.option(
+    "-h",
+    "--host",
+    type=str,
+    default=DEFAULT_RUNSERVER_HOST,
+    help=(
+        "Serve the LRS-compatible API on this host in the container "
+        f"(default {DEFAULT_RUNSERVER_HOST})"
+    ),
+)
+@click.option(
+    "-p",
+    "--port",
+    type=int,
+    default=DEFAULT_RUNSERVER_PORT,
+    help=f"Serve the LRS-compatible API on this port (default {DEFAULT_RUNSERVER_PORT})",
+)
+@click.option(
+    "-f",
+    "--auth-file",
+    type=str,
+    default="credentials/users.json",
+    help="File containing authentification & authorization information for API calls",
+)
+def runserver(host, port, auth_file):
+    """
+    Run the API server for the development environment. Starting uvicorn programmatically for
+    convenience and documentation.
+    """
+    logger.info(f"Running API server on {host}:{port}.")
+    logger.info(
+        (
+            "Do not use runserver in production - start production servers "
+            "through a process manager such as gunicorn/supervisor/circus."
+        )
+    )
+    try:
+        # NB: we must pass the live app in the same thread to uvicorn.run so it can access
+        # the current click context and make use of CLI parameters.
+        # This also prevents us from enabling live-reloading.
```

Comment:
This should no longer be the case, right?

Reply:
Yes, this is not necessary if we do not need to access CLI params from inside the server. For now there was only `auth_file` so we can use an import string dans enable auto reloading 👍